src/mmg3d/hash_3d.c

```diff
@@ -2002,6 +2002,7 @@ int MMG5_bdrySet(MMG5_pMesh mesh) {
     for (k=1; k<=mesh->ne; k++) {
       pt = &mesh->tetra[k];
       if ( !MG_EOK(pt) )  continue;
+      if (pt->tag & MG_OVERLAP) continue;
```

Comment:
For all this kind of tests, for now I am not able to say if they are valid, but:
  - It should not impact Mmg;
  -  I trust you that it is what we want to do in ParMmg ;-).